app/app.go

```diff
@@ -327,7 +352,7 @@ func NewSommelierApp(
 		),
 	)
 
-	app.AuthzKeeper = authzkeeper.NewKeeper(keys[authzkeeper.StoreKey], appCodec, app.BaseApp.MsgServiceRouter())
+	app.AuthzKeeper = authzkeeper.NewKeeper(keys[authzkeeper.StoreKey], appCodec, app.MsgServiceRouter())
```

Comment:
Out of curiosity, what was the need for this change?

Reply:
They refer to the same thing, and I found it confusing for `app.go` to have several different ways of referencing the same `MsgServiceRouter`.